app/src/main/java/ru/vssemikoz/newsfeed/models/Params.java

```diff
@@ -0,0 +1,49 @@
+package ru.vssemikoz.newsfeed.models;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Params {
```

Comment:
What will happen if we'll add another domain model like `Subscription` or `User`? Is `Params` a good and self-explained name for this class?